adr/0002-store-attestations-in-archivista.md

```diff
@@ -0,0 +1,29 @@
+# 1. Store In-Toto Attestations in Archivista
+
+Date: 2024-07-09
+
+## Status
+
+Accepted
+
+## Context
+
+During execution of Software Factory pipelines, we will be generating [in-toto attestations](https://github.com/in-toto/attestation) using [Witness](https://github.com/in-toto/witness) for the artifacts that are produced. We need to a location to store these attestations so that they can later be retrieved and used to verify an artifact against policies.
```

Comment:
```suggestion
During execution of Software Factory pipelines, we will be generating [in-toto attestations](https://github.com/in-toto/attestation) using [Witness](https://github.com/in-toto/witness) for the artifacts that are produced. We need a location to store these attestations so that they can later be retrieved and used to verify an artifact against policies.
```